### PR TITLE
Update zero-hour-auto-purge.md

### PR DESCRIPTION
**Pull request by andypunt**
Highlight the cons of using Safe Senders.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/zero-hour-auto-purge.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Microsoft 365 updates spam and malware signatures in real-time on a daily basis.
 
 The ZAP action is seamless for the user; they aren't notified if a message is detected and moved.
 
-[Safe sender lists](create-safe-sender-lists-in-office-365.md), mail flow rules (also known as transport rules), Inbox rules, or additional filters take precedence over ZAP.
+[Safe sender lists](create-safe-sender-lists-in-office-365.md), mail flow rules (also known as transport rules), Inbox rules, or additional filters take precedence over ZAP. Similar to what happens in mailflow, this means that even if the service determines it needs to Zap the message, the message will not be due to the safe senders configuration. It is for this reason that admins should be careful about what is being allowed to bypass our filters.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/zero-hour-auto-purge.md]:
```suggestion
[Safe sender lists](create-safe-sender-lists-in-office-365.md), mail flow rules (also known as transport rules), Inbox rules, or additional filters take precedence over ZAP. Similar to what happens in mail flow, this means that even if the service determines it needs to ZAP the message, the message will not be due to the safe senders configuration. It is for this reason that admins should be careful about what is being allowed to bypass our filters.
```

File: microsoft-365/security/office-365-security/zero-hour-auto-purge.md [hunk 2]
```diff
@@ -89,9 +89,9 @@ A: You should follow the normal reporting process for [false positives](report-j
 
 A: ZAP will take action on a message based on the configuration your anti-spam policies as described earlier in this topic.
 
-### Q: What if I'm using mail flow rules or allowed/blocked sender lists?
+### Q: What if I'm using Safe Senders, mail flow rules or allowed/blocked sender lists?
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/zero-hour-auto-purge.md]:
```suggestion
### Q: What if I'm using safe senders, mail flow rules, or allowed/blocked sender lists?
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@chrisda - PR has been copy edited and is ready for final review.

Thank you.

cc: @kenwith

**Pull request merged**